src/Makefile

```diff
@@ -61,17 +61,27 @@ config-gen: update
 config: config-i386
 
 config-linux: config-gen
```

Comment:
Should we keep `config-linux`? I'd think no.

Reply:
@phanikishoreg you need to discuss with @gparmer about this I think.